[2025-05-17T09:31:33.483+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: uhi_index_analytics_pipeline_decompose.process_nyzd manual__2025-05-17T05:12:03.413965+00:00 [queued]>
[2025-05-17T09:31:33.697+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: uhi_index_analytics_pipeline_decompose.process_nyzd manual__2025-05-17T05:12:03.413965+00:00 [queued]>
[2025-05-17T09:31:33.715+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-05-17T09:31:33.728+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-05-17T09:31:33.744+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-05-17T09:31:33.904+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): process_nyzd> on 2025-05-17 05:12:03.413965+00:00
[2025-05-17T09:31:34.122+0000] {standard_task_runner.py:55} INFO - Started process 1175 to run task
[2025-05-17T09:31:34.254+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'uhi_index_analytics_pipeline_decompose', 'process_nyzd', 'manual__2025-05-17T05:12:03.413965+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/uhi_pipeline_dag_decompose.py', '--cfg-path', '/tmp/tmptcx2e_uk']
[2025-05-17T09:31:34.257+0000] {standard_task_runner.py:83} INFO - Job 212: Subtask process_nyzd
[2025-05-17T09:31:35.360+0000] {task_command.py:388} INFO - Running <TaskInstance: uhi_index_analytics_pipeline_decompose.process_nyzd manual__2025-05-17T05:12:03.413965+00:00 [running]> on host ee2bb044a094
[2025-05-17T09:31:35.223+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uhi_index_analytics_pipeline_decompose
AIRFLOW_CTX_TASK_ID=process_nyzd
AIRFLOW_CTX_EXECUTION_DATE=2025-05-17T05:12:03.413965+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-17T05:12:03.413965+00:00
[2025-05-17T09:31:35.347+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/uhi_pipeline_dag_decompose.py", line 109, in process_nyzd_data
    preprocess.input_data_handle(data["nyzd"], type="nyzd")
TypeError: 'NoneType' object is not subscriptable
[2025-05-17T09:31:33.998+0000] {taskinstance.py:1323} INFO - Marking task as UP_FOR_RETRY. dag_id=uhi_index_analytics_pipeline_decompose, task_id=process_nyzd, execution_date=20250517T051203, start_date=20250517T093133, end_date=20250517T093133
[2025-05-17T09:31:34.114+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 212 for task process_nyzd ('NoneType' object is not subscriptable; 1175)
[2025-05-17T09:31:34.319+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-05-17T09:31:34.605+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
